documentation/src/main/asciidoc/public/reference/_search-dsl-predicate.adoc

```diff
@@ -1586,24 +1600,24 @@ include::{sourcedir}/org/hibernate/search/documentation/search/predicate/Predica
 ----
 ====
 
-The Elasticsearch backend, when using <<backend-elasticsearch-compatibility-elasticsearch,Elasticsearch>>,
-also allows configuring a backend-specific `knn` predicate option: the number of candidates.
-This option specifies a number of approximate nearest neighbor candidates to be found on each shard,
-then the results from each shard are merged and the top `k` are selected.
-When not specified explicitly, Hibernate Search will default the number of candidates value to `k`.
-See the Elasticsearch link:{elasticsearchDocUrl}/knn-search.html#tune-approximate-knn-for-speed-accuracy[documentation] for more details.
-<<backend-elasticsearch-compatibility-opensearch,OpenSearch>> does not expose this option.
-
-.Setting a number of candidates Elasticsearch-specific knn option
-====
-[source, JAVA, indent=0, subs="+callouts"]
-----
-include::{sourcedir}/org/hibernate/search/documentation/search/predicate/PredicateDslIT.java[tags=knn-candidates]
-----
-<1> Get an extended, Elasticsearch-specific, predicate factory.
-<2> Build a `knn` predicate as ususal.
-<3> Provide an Elasticsearch-specific predicate option.
-====
+// The Elasticsearch backend, when using <<backend-elasticsearch-compatibility-elasticsearch,Elasticsearch>>,
+// also allows configuring a backend-specific `knn` predicate option: the number of candidates.
+// This option specifies a number of approximate nearest neighbor candidates to be found on each shard,
+// then the results from each shard are merged and the top `k` are selected.
+// When not specified explicitly, Hibernate Search will default the number of candidates value to `k`.
+// See the Elasticsearch link:{elasticsearchDocUrl}/knn-search.html#tune-approximate-knn-for-speed-accuracy[documentation] for more details.
+// <<backend-elasticsearch-compatibility-opensearch,OpenSearch>> does not expose this option.
+//
+// .Setting a number of candidates Elasticsearch-specific knn option
+// ====
+// [source, JAVA, indent=0, subs="+callouts"]
+// ----
+// include::{sourcedir}/org/hibernate/search/documentation/search/predicate/PredicateDslIT.java[tags=knn-candidates]
+// ----
+// <1> Get an extended, Elasticsearch-specific, predicate factory.
+// <2> Build a `knn` predicate as ususal.
+// <3> Provide an Elasticsearch-specific predicate option.
+// ====
```

Comment:
You can delete this: commenting out code is so 2000s, the git history allows us to retrieve delete stuff nowadays ;D

Reply:
WHAT? Git can do that??? nooooooo wayyy!!! 😄 😆 
yeah, sorry 🙈  I was going to copy some of the links from this section before removing it, but never got to actually remove it 🙈